cli/cmd/vulnerability.go

```diff
@@ -392,6 +392,8 @@ func checkScanStatus(requestID string) (*api.VulnContainerAssessment, error, boo
 	switch status {
 	case "Success":
 		return &scan.Data, nil, false
+	case "Partial":
+		return &scan.Data, nil, false
```

Comment:
Should "scanning" be returned as true. Do we return the "Partial" result?

Reply:
Return partial results